src/main/java/QueryServlet.java

```diff
@@ -0,0 +1,68 @@
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.util.Map;
+import java.util.HashMap;
+import java.util.stream.Stream;
+import java.util.stream.Collectors;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+
+import com.google.gson.Gson;
+
+/** Servlet that handles census queries from users */
+@WebServlet("/query")
+public class QueryServlet extends HttpServlet {
+
+  // Hardcoded initial choices for which lines of the data table are accessed
+  Map<String, String> lineSelector = 
+    Stream.of(
```

Comment:
Optional: Maybe cleaner to use enums instead.

Reply:
Hmm. I agree that I think that would be better for now, but later I am envisioning that there will be a much larger map (or some other even more complex data structure, perhaps?) that is being used to match parts of queries to data tables. Do you think it's better to use an enum now knowing we will switch to something else later, or to leave it like this?

Reply:
Resolving. If we do end up using just the string maybe we can revisit it.